data/core/macros/weapon_specials.cfg

```diff
@@ -145,19 +145,32 @@
     [/damage]
 #enddef
 
-#define WEAPON_SPECIAL_ABSORB AMT
+#define WEAPON_SPECIAL_ABSORB_40
     # Canned definition of the Absorb ability to be included in a
     # [specials] clause.
     [damage]
-        id=absorb
-        name= _ "absorb"
-        description= _ "This attack puts the unit in good defensive position, and it absorbs some of the damage dealt by an enemy strike."
+        id=absorb_40
+        name= _ "absorb 40%"
+        description= _ "This attack puts the unit in good defensive position, and it absorbs 40% of the damage dealt by an enemy strike."
         special_note= _ "This unit can block enemy strikes, so that it takes reduced damage when hit."
-        multiply={AMT}
+        multiply=0.6
         apply_to=opponent
     [/damage]
 #enddef
 
+#define WEAPON_SPECIAL_ABSORB_25
+# Canned definition of the Absorb ability to be included in a
```

Comment:
This section should be indented in the same manner as the #definition above, to satisfy the CI check.

Reply:
Should be good now.